samples/test-samples.cmake

```diff
@@ -50,7 +65,7 @@ if (BUILD_ENCLAVES)
     list(APPEND CRYPTO_LIB_LIST openssl_3)
   endif ()
 
-  if (UNIX)
+  if (UNIX AND NOT IS_JAMMY)
```

Comment:
Requested change: comment to explain why this is not supported on Jammy

Reply:
Done